src/researches/passiveVoice/periphrastic/determinePassiveSentencePart.js

```diff
@@ -0,0 +1,35 @@
+const arrayToRegex = require( "../../../stringProcessing/createRegexFromArray.js" );
+const determineSentencePartIsPassive = require( "./determineSentencePartIsPassive.js" );
+
+// Auxiliaries (only needed for German)
+const auxiliariesGerman = require( "../../german/passiveVoice/auxiliaries.js" )().allAuxiliaries;
+
+// Participles
+const getParticiplesGerman = require( "../../german/passiveVoice/getParticiples.js" );
+const getParticiples = require( "./getParticiples.js" );
+
+
+/**
+ * Determines whether a sentence part is passive.
+ *
+ * @param {string} sentencePartText The sentence part to determine voice for.
+ * @param {Array} auxiliaries A list with auxiliaries in this sentence part.
+ * @param {string} language The language of the sentence part.
+
+ * @returns {boolean} Returns true if passive, otherwise returns false.
+ */
+module.exports = function( sentencePartText, auxiliaries, language ) {
+	let participles = [];
+	if ( language === "de" ) {
+		let passive = false;
+		let auxiliaryMatches = sentencePartText.match( arrayToRegex( auxiliariesGerman ) );
+		if ( auxiliaryMatches === null ) {
+			return passive;
+		}
+		participles = getParticiplesGerman( sentencePartText, auxiliaries, language );
```

Comment:
Are the German Participles also different, the auxiliaries have a comment about being an exclusion, but nothing tells me the participles are that as well.

Replacing the auxiliaries in the "de" conditional and then doing a general participles call looks to be a better solution, unless some context is missing that would describe that this is not the case.

Reply:
German participles are also defined differently than the other. I edited the comments to make it explicit.